src/openfl/_internal/renderer/canvas/CanvasShape.hx

```diff
@@ -44,23 +44,18 @@ class CanvasShape {
 					context.globalAlpha = shape.__worldAlpha;
 					
 					var transform = graphics.__worldTransform;
+					var pixelRatio = renderSession.pixelRatio;
```

Comment:
I could imagine a `var scale = 1;` here which is used in the calculation below for `transform.a` and `transform.b`. I believe it will be optimized away in JS, but it somehow highlights that it wasn't just forgotton here. We could also comment why this is always 1, but because comments are kind of not used in OpenFL, I would just go for the `scale = 1` way

Reply:
Hmm yeah we discussed that before, maybe makes sense.